charts/addons/values.yaml

```diff
@@ -33,24 +33,24 @@ traefik:
       # Enter the memory limit for Traefik
       memory: 500Mi
 
-# When deploying on AWS EKS, set this to true to install aws-alb-ingress-controller.
-aws-alb-ingress-controller:
+# When deploying on AWS EKS, set this to true to install aws-load-balancer-controller.
+aws-load-balancer-controller:
   enabled: false
   ## Resources created by the ALB Ingress controller will be prefixed with this string
   clusterName: "YOUR_EKS_CLUSTER_NAME"
   ## Auto Discover awsRegion from ec2metadata, set this to true and omit awsRegion when ec2metadata is available.
   autoDiscoverAwsRegion: false
   ## AWS region of k8s cluster, required if ec2metadata is unavailable from controller pod
   ## Required if autoDiscoverAwsRegion != true
-  awsRegion: "YOUR_EKS_CLUSTER_REGION"
-  ## Auto Discover awsVpcID from ec2metadata, set this to true and omit awsVpcID: " when ec2metadata is available.
-  autoDiscoverAwsVpcID: false
```

Comment:
Is this stopped supporting in latest chart?

Reply:
yes, it is not supported/used - https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/values.yaml
I just realized autoDiscoverAwsRegion is also not available in the latest chart, I'll remove that as well.

Reply:
hi - when you update the values file, can you please up date the annotation to something like this?

# When deploying on AWS EKS, set the parameters, install aws-alb-ingress-controller and metrics-server
# to true and then fill in the remaining parameters with your EKS environment details.

please let me know when you are ready and I can review the doc for values.yaml and the readme.md for addons ;-)

Reply:
Sure @taz-mon I've updated README and addons values.yaml. Please review.